frontend/src/components/user-details/components/user-profile/components/user-profile-form/user-profile-form.tsx

```diff
@@ -44,6 +46,9 @@ const UserProfileForm: FC = () => {
       reset({
         fullName: userDetails.fullName,
         gender: userDetails.gender ?? UserGender.MALE,
+        dateOfBirth: userDetails.dateOfBirth
+          ? (new Date(userDetails.dateOfBirth) as unknown as string)
```

Comment:
```suggestion
          ? new Date(userDetails.dateOfBirth).toString()
```
🤔

Reply:
@what1s1ove 
![image](https://user-images.githubusercontent.com/78655088/187662735-8cf15582-8a48-4056-877e-e9d6e9ed0f3d.png)
I get this error if I try anything else except `as unknown as string`

Reply:
in this case, you need to provide `Date`, instead of string